modules/oadp-release-notes-1-1-2.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes.adoc
+:_content-type: REFERENCE
+[id="migration-oadp-release-notes-1-1-2_{context}"]
+= OADP 1.1.2 release notes
+
+The OADP 1.1.2 release notes include product recommendations.
+
+== Product recommendations
+
+Before you install OADP 1.1.2, it is recommended to either install VolSync 0.5.1 or to upgrade to it.
+
+To prepare for the upgrade from VolSync 0.5.1 to 0.6.0, in the `openshift-adp` namespace you must add this annotation.
+
+----
+`oc annotate --overwrite namespace/openshift-adp volsync.backube/privileged-movers='true'`
```

Comment:
```suggestion
$ oc annotate --overwrite namespace/openshift-adp volsync.backube/privileged-movers='true'
```

Reply:
Does  the value `true` need to be in italics?

Reply:
No, sorry I don't remember when I got change to italics